### PR TITLE
Modules 07: no VPN references in src/shared

### PR DESCRIPTION
**Pull request by bakulf**
I missed a few things.

### CODE REVIEW

File: src/apps/template/appconstants.h [hunk 1]
```diff
@@ -22,6 +22,33 @@ constexpr const char* MACOS_FALLBACK_APP_ID = "org.mozilla.macos.AppFooBar";
 constexpr const char* IOS_FALLBACK_APP_ID = "org.mozilla.ios.AppFooBar";
 #endif
 
+#if defined(MZ_WINDOWS)
```
Comment (oskirby) [inline src/apps/template/appconstants.h:25]: I feel like these should not be in the template unless they are things we expect *all* applications to provide, and I don't think it's appropriate to assume that all applications are going to support addons and cryptosettings.

File: src/apps/template/appconstants.h [hunk 2]
```diff
@@ -22,6 +22,33 @@ constexpr const char* MACOS_FALLBACK_APP_ID = "org.mozilla.macos.AppFooBar";
...
+// Credential key for windows
+constexpr const wchar_t* WINDOWS_CRED_KEY = L"Foo bar";
+#endif
+
+#if defined(MZ_ANDROID)
+constexpr const char* ANDROID_LOG_NAME = "mozillavpn";
```
Comment (oskirby) [inline src/apps/template/appconstants.h:31]: `constexpr const char* ANDROID_LOG_NAME = "foobar";` ?

File: src/apps/template/appconstants.h (hunk 1 above)
Comment (bakulf) [reply src/apps/template/appconstants.h:25]: Why not? Why we should have not encrypted settings? Plus add-ons are a key component for content delivering. The suite app needs both components.

Comment (oskirby) [reply src/apps/template/appconstants.h:25]: I'm just thinking ahead for stuff like unit tests where we may not want to introduce platform-specific variability in the test runs.

Comment (bakulf) [reply src/apps/template/appconstants.h:25]: Why not? We run unit tests on all the platforms. In this way we increase the test coverage.

**Review by oskirby — APPROVED** [review]

**Pull request merged**